src/main/features/ccj/routes/check-and-send.ts

```diff
@@ -18,28 +22,69 @@ function prepareUrls (externalId: string): object {
 }
 
 function renderView (form: Form<Declaration>, req: express.Request, res: express.Response): void {
+  const user: User = res.locals.user
   res.render(Paths.checkAndSendPage.associatedView, {
     form: form,
-    details: res.locals.user.ccjDraft,
-    amountToBePaid: res.locals.user.claim.totalAmount - (res.locals.user.ccjDraft.paidAmount.amount || 0),
+    details: user.ccjDraft,
+    amountToBePaid: user.claim.totalAmount - (user.ccjDraft.paidAmount.amount || 0),
+    partyAsCompanyOrOrganisation: isPartyCompanyOrOrganisation(user),
     ...prepareUrls(req.params.externalId)
   })
 }
 
+function isPartyCompanyOrOrganisation (user: User): boolean {
```

Comment:
I can see something similar in claim check and send, maybe it would be worth to extract a common component?

Reply:
it looks similar but it looks somewhere else for it the data.
all the functions are similar but slightly different :(

Reply:
I would move this method to party class so we that we have this method available for any party, no matter what side of the fence their are. Having this we could do `if (claim.claimData.claimant.isBusiness())`. I am not strong on the name of this method, could be something different then I proposed.